dashboard/app/controllers/admin_reports_controller.rb

```diff
@@ -142,7 +142,7 @@ def pd_progress
         )
         render(
           layout: 'application',
-          text: "PD progress data not found for #{sanitized_script_name}.",
+          plain: "PD progress data not found for #{sanitized_script_name}.",
```

Comment:
These two seem worth verifying before merging since they also specify a layout.

Reply:
oh wow, nice catch; I literally looked right at these and didn't see a `layout` :upside_down_face: